README.md

```diff
@@ -0,0 +1,53 @@
+# CrateDB tableau's connector.
+
+For simple queries and graphs, you can use the default PostgresSQL connector,
+but for more advance usage SQL compatibility issues and nuanced SQL functionalities get into play,
+we circumvent/fix those limitations with our custom connector.
+
+## How to use.
+
+You will need two things:
+
+The `postgresql jdbc driver` and the `cratedb_jdbc`
```

Comment:
The `cratedb_jdbc` label might be confused with the legacy CrateDB JDBC driver https://github.com/crate/crate-jdbc for those who know it. Is there a chance to name it differently?

Reply:
Yeah, we can rename it, any suggestion? perhaps `cratedb_tableau_jdbc_connector` ? we shouldn't omit `jdbc` since there is also the option of making an `odbc` connector,

Reply:
> we shouldn't omit `jdbc` since there is also the option of making an `odbc` connector.

I see. Expected that. Thanks.

> perhaps `cratedb_tableau_jdbc_connector`? 

`cratedb_tableau_jdbc_connector` is definitively a long label, but optimally conveys what's inside. I don't have any better suggestion. Thanks!